apps/prairielearn/src/lib/code-caller/code-caller-native.ts

```diff
@@ -118,7 +118,7 @@ export class CodeCallerNative implements CodeCaller {
   static async create(options: CodeCallerNativeOptions): Promise<CodeCallerNative> {
     const pythonExecutable = await run(async () => {
       for (const p of options.pythonVenvSearchPaths) {
-        const venvPython = path.resolve(REPOSITORY_ROOT_PATH, path.join(p, 'bin', 'python3.10'));
+        const venvPython = path.resolve(REPOSITORY_ROOT_PATH, path.join(p, 'bin', 'python3.13'));
```

Comment:
Aside, I see this committed in many PRs of yours. Can We do anything to make this less likely to get committed for you? e.g. make it configurable? How do you even end up using 3.13 given that the `venv-setup` Make target explicitly uses 3.10 with `uv`?

Reply:
@jonatanschroeder I have had trouble with that approach every time there is a change I didn't make to `apps/prairielearn/src/lib/code-caller/code-caller-native.ts` (i.e. merge via master, I have edited the file locally and added it to that list). Git gets mad at me when that happens.